analyzer-models/src/main/java/ffb/analyzer/models/espn/TeamRecordBase.java

```diff
@@ -16,12 +16,17 @@
 
     private int gamesBack;
 
-    @JsonProperty("percentage")
-    private float winPercentage;
-
     private int streakLength;
     private StreakType streakType;
 
+    public float getWinPercentage() {
+        return (float) wins / (float) getCountGamesPlayed();
```

Comment:
Should we cache this calculation?

Reply:
Yep, this one makes sense too since we're aggregating and performing type conversion. I considered caching getCountGamesPlayed as well, but it seems like a simpler calculation that wouldn't benefit as much.